include/Song.h

```diff
@@ -158,7 +129,7 @@ class LMMS_EXPORT Song : public TrackContainer
 	inline void setToTime(TimePos const & pos, PlayMode playMode)
 	{
 		m_elapsedMilliSeconds[static_cast<std::size_t>(playMode)] = pos.getTimeInMilliseconds(getTempo());
-		getPlayPos(playMode).setTicks(pos.getTicks());
+		getTimeline(playMode).setTicks(pos.getTicks());
```

Comment:
`m_elapsedMilliseconds` is returned in `Song::getMillisends`, and afaik is only when the current time lcd is being updated. This could be removed and instead be calculated on the fly using the functions in `TimePos`, but if the tempo is changed during the song, this will not give the correct result.

`m_elapsedBars` and `m_elapsedTicks` do not appear to be used anywhere in the codebase, so I suppose they can be deleted?